packages/bitwuzla/bitwuzla.1.0.2/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+synopsis: "SMT solver for AUFBVFP"
+description: """
+
+OCaml binding for the SMT solver Bitwuzla.
+
+Bitwuzla is a Satisfiability Modulo Theories (SMT) solver for the theories of fixed-size bit-vectors, arrays and uninterpreted functions and their combinations. Its name is derived from an Austrian dialect expression that can be translated as “someone who tinkers with bits”."""
+maintainer: ["Frédéric Recoules <frederic.recoules@cea.fr>"]
+authors: ["Frédéric Recoules"]
+license: "MIT"
+tags: ["SMT solver" "SMT-COMP 2020" "AUFBVFP"]
+homepage: "https://bitwuzla.github.io"
+doc: "https://bitwuzla.github.io/docs/ocaml/"
+bug-reports: "https://github.com/bitwuzla/ocaml-bitwuzla/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "bitwuzla-c"
```

Comment:
I think you need
```suggestion
  "bitwuzla-c" {>= "1.0.2"}
```
for the error below. But I would really recommend to do
```suggestion
  "bitwuzla-c" {= version}
```
and always keep the releases tied together to avoid this kind of problem.

The error is
```
#=== ERROR while compiling bitwuzla.1.0.2 =====================================#
# context              2.1.2 | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/bitwuzla/ocaml-bitwuzla/releases/download/1.0.2/bitwuzla-1.0.2.tbz)
# path                 ~/.opam/4.08/.opam-switch/build/bitwuzla.1.0.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p bitwuzla -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/bitwuzla-2899-96de7c.env
# output-file          ~/.opam/log/bitwuzla-2899-96de7c.out
### output ###
#     ocamlopt .bitwuzla.objs/native/bitwuzla.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlopt.opt -w -40 -g -I .bitwuzla.objs/byte -I .bitwuzla.objs/native -I /home/opam/.opam/4.08/lib/bitwuzla-c -I /home/opam/.opam/4.08/lib/zarith -intf-suffix .ml -no-alias-deps -o .bitwuzla.objs/native/bitwuzla.cmx -c -impl bitwuzla.ml)
# File "bitwuzla.ml", line 635, characters 8-13:
# 635 |       | Const ->
#               ^^^^^
# Error: This variant pattern is expected to have type Bitwuzla_c.kind
#        The constructor Const does not belong to type Bitwuzla_c.kind
#       ocamlc .bitwuzla.objs/byte/bitwuzla.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlc.opt -w -40 -g -bin-annot -I .bitwuzla.objs/byte -I /home/opam/.opam/4.08/lib/bitwuzla-c -I /home/opam/.opam/4.08/lib/zarith -intf-suffix .ml -no-alias-deps -o .bitwuzla.objs/byte/bitwuzla.cmo -c -impl bitwuzla.ml)
# File "bitwuzla.ml", line 635, characters 8-13:
# 635 |       | Const ->
#               ^^^^^
# Error: This variant pattern is expected to have type Bitwuzla_c.kind
#        The constructor Const does not belong to type Bitwuzla_c.kind
```

Reply:
Thank you, I applied your suggestion ([1570bc5](https://github.com/bitwuzla/ocaml-bitwuzla/commit/1570bc5cd812811cd7ad37eb0d16775737222673))